CHANGELOG.md

```diff
@@ -23,6 +23,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Thank you to all who have contributed!
 -->
 
+## [Unreleased](https://TODO.com) - YYYY-MM-DD
+
+### Added
+
+### Changed
+
+### Deprecated
+
+### Fixed
+
+### Removed
+
+### Security
+
+### Contributors
+Thank you to all who have contributed!
+
+## [1.2.0](https://github.com/partiql/partiql-lang-kotlin/releases/tag/v1.2.0) - 2025-03-12
+
+### Added
+- Added parsing, planning, and execution of the interval type and interval values.
+- Added multiple operations involving datetime and interval values.
+- Added planning and evaluation of the LET clause
```

Comment:
(side-comment) this PR probably needs to be rebased off of `main`. currently, this PR is showing changes that I assume were not yet in `main` (e.g. interval, let clause) at the time the PR was created, leading to a larger than expected diff